handlebars-cmd/plan.sh

```diff
@@ -0,0 +1,62 @@
+pkg_name=handlebars-cmd
+pkg_origin=core
+pkg_version="0.1.4"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('MIT')
+pkg_source=nosuchfile.tar.gz
+pkg_deps=(
+  core/coreutils
+  core/node
+)
+pkg_build_deps=(
+  core/coreutils
+  core/node
+)
+pkg_description="handlebars command-line tool. Shrinkwrapped with handlebars 4.0.6."
+pkg_upstream_url="https://github.com/spion/handlebars-cmd"
+pkg_bin_dirs=(bin)
+pkg_interpreters=(bin/env)
+
+do_prepare() {
+  # For the benefit of ncu.
+  if [[ ! -r /usr/bin/env ]]; then
+    ln -svf "$(pkg_path_for coreutils)/bin/env" /usr/bin/env
+     _clean_env=true
+  fi
+
+  npm config set progress false
+}
+
+do_download() {
+  return 0
+}
+
+do_verify() {
+  return 0
+}
+
+do_unpack() {
+  return 0
+}
+
+do_build() {
+  return 0
+}
+
+do_install() {
+  npm install -g handlebars-cmd
+  npm install -g npm-check-updates
+  pushd "$pkg_prefix/lib/node_modules/handlebars-cmd"
```

Comment:
we've usually been putting `> /dev/null` on the `pushd` and `popd` commands to silence them.